web/themes/custom/pineapple/components/_hero-image.twig

```diff
@@ -1,40 +1,45 @@
 {% set content_url = content_url|default('') %}
-<div class="hero-container breakout">
+{% set hero_attributes = hero_attributes|default(create_attribute()).addClass([
+  'hero-container',
+  'breakout'
+]) %}
+<div{{ hero_attributes }}>
   {% if content_url %}
     <a href="{{ content_url }}">
   {% endif %}
```

Comment:
While here update this to work like the url_prefix in the paragraph image grid file

Reply:
Okay, done.